Samples/Samples/View/CalendarEventPage.xaml.cs

```diff
@@ -86,29 +84,22 @@ async void OnRemoveAttendeeFromEventButtonClicked(object sender, EventArgs e)
                     {
                         lst.Remove(attendeeToRemove);
                     }
-                    BindingContext = new CalendarEvent()
-                    {
-                        AllDay = ViewModel.AllDay,
-                        Attendees = lst,
-                        CalendarId = ViewModel.CalendarId,
-                        Description = ViewModel.Description,
-                        Duration = ViewModel.Duration,
-                        EndDate = ViewModel.EndDate,
-                        Id = ViewModel.Id,
-                        Location = ViewModel.Location,
-                        StartDate = ViewModel.StartDate,
-                        Title = ViewModel.Title
-                    };
+                    ViewModel.Attendees = lst;
```

Comment:
Investigate why we're manually updating bindingcontext instead of just binding to the viewmodel

Reply:
Altered to use viewmodel instead of binding to API model